docs/api.md

```diff
@@ -725,6 +735,40 @@ Determines whether or not boss bar creates fog
 
 Determines what color the boss bar color is, one of `pink`, `blue`, `red`, `green`, `yellow`, `purple`, `white`
 
+### mineflayer.Particle
+
+#### Particle.id
+
+Particle ID, as defined in the [protocol](https://wiki.vg/Protocol#Particle)
+
+#### Particle.name
+
+Particle Name, as defined in the [protocol](https://wiki.vg/Protocol#Particle)
+
+#### Particle.longDistance
+
+Determines whether or not to increase the particle distance
+
+#### Particle.position
+
+Vec3 instance of where the particle was created
+
+#### Particle.offset
+
+Vec3 instance of the particle's offset
+
+#### Particle.particleData
```

Comment:
This type of data should be abstracted away, not exposed directly. It's likely to break between versions and between different editions of Minecraft (eg bedrock). Try to only expose data/fields that would make sense to an ingame player (high level).

Reply:
This could be changed to `maxSpeed` as defined in the latest protocol however I wasn't sure if it had any other use in previous versions as it was originally called `particleData`.

Reply:
yeah, would be a good idea to check this then. Users shouldn't have to do it on their side

Reply:
Checked a few Minecraft version's source code, from what I can tell it has no other use other than determining the speed to move a particle in a random direction, changed accordingly.